website/content/about/CloudNativeSustainabilityWeek.md

```diff
@@ -0,0 +1,80 @@
+| 23 | Aydin Mir Mohammadi | Germany, Karlsruhe | *tbd* | *tbd* |
+| 24 | Leonard Pahlke | Germany, Hamburg | *tbd* | *tbd* |
+| 25 | Michel Murabito | Italy, Milan | *tbd* | *tbd* |
+
+## Event Goals
+
+1. **Understand the current [landscape](https://tag-env-sustainability.cncf.io/about/landscape/)**: Engaging with the community and industry to better understand the [landscape](https://tag-env-sustainability.cncf.io/about/landscape/) of cloud native environmental sustainability.
+2. **Identify Knowledge Gaps**: Identifying areas of knowledge gaps and further exploration in cloud native environmental sustainability.
+3. **Foster project collaboration**: Facilitating collaboration among participants on ongoing projects and tools related to cloud native sustainability.
+4. **Raise awareness**: Increasing awareness and knowledge about cloud native environmental sustainability.
+5. **Promote TAG ENV and CNCF**: Promoting the #tag-environmental-sustainability and raising awareness about the CNCF in general.
+6. **Survey**: distributing a survey to gather information on end user adoption of cloud native sustainability.
+7. **Badges**: providing badges from credly for organizers and potentially speakers.
+
+This [document](https://docs.google.com/document/d/1s28lqqc3IBAMw4T7n13zfyAuUEEc3I4_xhPxCLA4_dk/edit?usp=sharing) describes the goals in more detail and the plan how we can achieve them.
+
+## Contact and Resources
+
+Please reach out to us via the [CNCF Slack workspace](https://slack.cncf.io/) in the channel [tag-env-sustainability](https://cloud-native.slack.com/archives/C03F270PDU6), if you have any comments or questions. All resources are publicly accessible.
+If you are a meetup organizer, we have a [tracking issue](https://github.com/cncf/tag-env-sustainability/issues/134) to collect input and questions.
+
+* **General Tracking Issue**: https://github.com/cncf/tag-env-sustainability/issues/95
+* **General Meetup Organizer Guide**: https://docs.google.com/document/d/1s28lqqc3IBAMw4T7n13zfyAuUEEc3I4_xhPxCLA4_dk/edit?usp=sharing 
+* **Milestone to track the event organization tasks**: https://github.com/cncf/tag-env-sustainability/milestone/3
+* **Google Drive folder**: https://drive.google.com/drive/folders/1pdBQTxlpvDr0QFIuIpjfERau1IQNfdaX?ths=true 
+
+## Acknowledgements ✨
+
+The event is organized by volunteers in their spare time. Its truly a community effort without that this would not be possible. Thanks everyone for helping!
+
+Special thanks to Nancy Chauhan, Adriana Villela, Kristina Devochko, Niki Manoledaki, Mercy Bamiduro, Mehmet, Rodolfo Martínez Vega and Leonard Pahlke to help organise the event.
+
+Special thanks to all the local meetup organizers.
+
+Special thanks to Katie Gamanji, Ricardo Rocha, Wojtek Cichoń, Nikhita Raghunath, Tom Kerkhove, Alena Prokharchyk, Abdel Sghiouar, Huamin Chen, Nate W. and Emily Fox for facilitating, sharing their thoughts and reaching out to people that might be interested organizing a meetup.
```

Comment:
Same again here.  Specific order?  Or should we do alphabetic?

Reply:
we can make it alphabetic. Good point!